includes/pull-ui.php

```diff
@@ -326,6 +330,52 @@ function( $remote_post_id ) use ( $post_type, $post_status ) {
 			// Redirect to the skipped content tab
 			wp_safe_redirect( add_query_arg( 'status', 'skipped', wp_get_referer() ) );
 			exit;
+		case 'bulk-unskip':
+		case 'unskip':
+			if ( ! wp_verify_nonce( $_GET['_wpnonce'], 'dt_unskip' ) && ! wp_verify_nonce( $_GET['_wpnonce'], 'bulk-distributor_page_pull' ) ) {
+				exit;
+			}
+
+			// Filter documented above.
+			if ( ! current_user_can( apply_filters( 'dt_pull_capabilities', 'manage_options' ) ) ) {
+				wp_die(
+					'<h1>' . esc_html__( 'Cheatin&#8217; uh?', 'distributor' ) . '</h1>' .
+					'<p>' . esc_html__( 'Sorry, you are not allowed to add this item.', 'distributor' ) . '</p>',
+					403
+				);
+			}
+
+			if ( empty( $_GET['connection_type'] ) || empty( $_GET['connection_id'] ) || empty( $_GET['post'] ) ) {
+				break;
+			}
+
+			if ( 'external' === $_GET['connection_type'] ) {
+				$connection = \Distributor\ExternalConnection::instantiate( intval( $_GET['connection_id'] ) );
+			} else {
+				$site       = get_site( intval( $_GET['connection_id'] ) );
+				$connection = new \Distributor\InternalConnections\NetworkSiteConnection( $site );
+			}
+
+			$posts = $_GET['post'];
+			if ( ! is_array( $posts ) ) {
+				$posts = [ $posts ];
+			}
+
+			$sync_log = $connection->get_sync_log( intval( $_GET['connection_id'] ) );
+
+			foreach ( $posts as $post_id ) {
+				if ( array_key_exists( $post_id, $sync_log ) ) {
+					unset( $sync_log[ $post_id ] );
+				}
+			}
+
+			$connection->log_sync( $sync_log, intval( $_GET['connection_id'] ), true );
+
+			setcookie( 'dt-unskipped', 1, time() + DAY_IN_SECONDS, ADMIN_COOKIE_PATH, COOKIE_DOMAIN, is_ssl() );
```

Comment:
@dkotter VIP will flag this due to caching, although as it's in the admin it may not be an issue. Testing locally on VIP seems fine, but it may fail PHPCS.

Reply:
Yeah, I debated changing how this works but this is how we already handle skipped or pulled items, by setting a cookie. So I decided it was best to just follow the pattern we already have in place.